base/utils/identity.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	"encoding/base64"
+	"encoding/json"
+)
+
+type Entitlement struct {
+	Entitled bool `json:"is_entitled"`
+}
+
+type IdentityDetail struct {
+	AccountNumber string `json:"account_number"`
+	Type          string `json:"type"`
+	// Additional information, we don't parse this
+	Internal      map[string]interface{}
+}
+
+type Identity struct {
+	Entitlements map[string]Entitlement `json:"entitlements"`
+	Identity     IdentityDetail         `json:"identity"`
+}
+
+func ParseIdentity(identityString string) (ident *Identity, err error) {
```

Comment:
Named return values are not used here. Let's use types only.

Reply:
Fixed